appcues/src/main/java/com/appcues/data/model/ExperienceStepFormState.kt

```diff
@@ -0,0 +1,167 @@
+package com.appcues.data.model
+
+import androidx.compose.runtime.mutableStateOf
+import com.appcues.data.model.ExperiencePrimitive.OptionSelectPrimitive
+import com.appcues.data.model.ExperiencePrimitive.TextInputPrimitive
+import com.appcues.data.model.ExperienceStepFormItemState.OptionSelectFormItemState
+import com.appcues.data.model.ExperienceStepFormItemState.TextInputFormItemState
+import com.appcues.data.model.styling.ComponentSelectMode.MULTIPLE
+import com.appcues.data.model.styling.ComponentSelectMode.SINGLE
+import java.util.UUID
+
+internal class ExperienceStepFormState {
```

Comment:
I'm not sure why this is showing as a new file rather than a move, perhaps due to some contents changes. I'll note the important parts that changed.